### DISCUSSION TITLE
Mycroft and AIY hat

### DISCUSSION THREAD

**Discussion by bwims** [MichaIng/DietPi#5788] [Q&A] [answered]
Hi,

Can anyone confirm that the Dietpi Mycroft installation will work with the Google AIY V1 hat?

According to the description of [Picroft](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/get-mycroft/picroft) , it is a tested piece of hardware (about two screens down from the link) but I'm not sure how much picroft maps on to the dietpi version.

**Comment by MichaIng** [comment, marked as answer]
It should work. Mycroft AI uses the regular ALSA interfaces, so the actual sound card shouldn't matter as long as microphone and playback generally works.

**Reply by bwims** [reply]
Thanks!

**Reply by bwims** [reply]
I suppose the next question is :  can I install the AIY hat as a sound card without installing the AIY software?  (I should have led with that)
The Picroft matrix implied that it would just work.

**Reply by MichaIng** [reply]
Yes the sound card driver ans ALSA config incl. playback and microphone (when selecting it via `dietpi-config` audio options) should work independent of the Google AIY software. Mycroft AI and Google AIY should be exchangeable both with any sound card.

But this is just based on reviewing how the sound card driver is enabled and the software install done + some logic, i.e. I haven't tested it and have no Google Voice HAT to test it 😉.

**Reply by bwims** [reply]
Many thanks for taking the extra time for the supplementary!

B.
